javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,90 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
```

Comment:
This has the same name as the const. 
That might be an issue.

Reply:
👍🏻